src/Database/helpers.js

```diff
@@ -0,0 +1,49 @@
+const Notice = require('./schema/notice');
+
+/*
+* noticeFromJSONDiff creates a notice object from a JSON diff
+* used for additions
+*/
+function noticeFromJSONDiff(jn) {
+  const n = new Notice();
+  n.geocoord = { type: 'Point', coordinates: [jn.data.lat, jn.data.long] };
+  n.objectid = jn.data.objectid;
+  n.title = jn.data.title;
+  n.status = jn.data.status;
+  n.worktype = jn.data.worktype;
+  n.affectedpremises = jn.data.affectedpremises;
+  n.trafficimplications = jn.data.trafficimplications;
+  n.description = jn.data.description;
+  n.globalid = jn.data.globalid;
+  n.location = jn.data.location;
+  n.startdate = jn.data.startdate;
+  n.enddate = jn.data.enddate;
+  n.county = jn.data.county;
+  n.referencenum = jn.data.referencenum;
+  n.noticetype = jn.data.noticetype;
+  n.contactdetails = jn.data.contactdetails;
+  return n;
+}
+
+/*
+* lowercaseKeys converts the keys of an object into lowercase
+*/
+function lowercaseKeys(obj) {
```

Comment:
Are the notices generally stored/transferred with lowercase keys? 
If so, we will need to update the app to expect lowercase keys too.

Reply:
Antti's schema uses lowercase keys, so most likely. I don't know if Mongo will let us store with uppercase keys, and I didn't want to mess with the schema more than I had to

Reply:
Okay thanks for the information. 
I will keep it as an agenda point for Thursday.